packages/adapter-drizzle/src/index.ts

```diff
@@ -71,22 +91,23 @@ import type { Adapter } from "@auth/core/adapters"
  *  integer
  * } from "drizzle-orm/pg-core"
  * import type { AdapterAccount } from '@auth/core/adapters'
+ * import { randomUUID } from "crypto"
  *
- * export const users = pgTable("user", {
- *  id: text("id").notNull().primaryKey(),
+ * export const users = pgTable("users", {
+ *  id: text("id").primaryKey().$defaultFn(() => randomUUID()),
  *  name: text("name"),
- *  email: text("email").notNull(),
+ *  email: text("email").unique(),
```

Comment:
not every provider has a mandatory email, for example `github`

Reply:
for GitHub, we still get the email if it's null https://github.com/nextauthjs/next-auth/blob/edc4fe4b4294fe6eed083f7acf641314722a4618/packages/core/src/providers/github.ts#L156-L163